recipes/sensor.rb

```diff
@@ -0,0 +1,61 @@
+# install and run the metaflows sensor/collector
+include_recipe 'runit'
+
+if node['platform'] == 'centos'
```

Comment:
Is not doing anything else the appropriate behavior? Or should an exception be thrown because someone's trying to run on an unsupported platform?